addon/components/o-s-s/skeleton.ts

```diff
@@ -0,0 +1,83 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+interface Row {
+  style?: string;
+}
+
+interface OSSSkeletonArgs {
+  width: number;
+  height: number;
+  multiple?: number;
+  type?: 'row' | 'column';
+  gap?: number;
+}
+
+export default class OSSSkeleton extends Component<OSSSkeletonArgs> {
+  constructor(owner: unknown, args: OSSSkeletonArgs) {
+    super(owner, args);
+
+    if (this.args.type) {
+      assert(
+        `[component][OSS::Skeleton] The @type argument should be a value of ${['row', 'column']}`,
+        ['row', 'column'].includes(this.args.type)
+      );
+    }
+  }
+
+  get height(): number {
+    return this.args.height || 100;
```

Comment:
default height should be something more reasonable, maybe `36px` to match  our input height ?